lab-sarah/model/client.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const uuid = require('node-uuid');
+
+const Client = module.exports = function(socket) {
```

Comment:
You have a linter error from this. Since you're not using the Client constructor inside this file, you don't need to name it at all!